.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  release:
+    types: [created]
```

Comment:
Why this instead of pushing a tag, like we do in other repositories?

Reply:
mostly because I was following the wombat release instructions, but I don't mind either way :)